sklearn/utils/_fast_dict.pyx

```diff
@@ -91,12 +87,10 @@ cdef class IntFloatDict:
                 The values of the data points
         """
         cdef int size = self.my_map.size()
-        cdef cnp.ndarray[ITYPE_t, ndim=1] keys = np.empty(size,
-                                                         dtype=np.intp)
-        cdef cnp.ndarray[DTYPE_t, ndim=1] values = np.empty(size,
-                                                           dtype=np.float64)
+        cdef ITYPE_t[:] keys = np.empty(size, dtype=np.intp)
+        cdef DTYPE_t[:] values = np.empty(size, dtype=np.float64)
         self._to_arrays(keys, values)
-        return keys, values
+        return np.asarray(keys), np.asarray(values)
```

Comment:
It doesn't work to just do the following ?
```
        keys = np.empty(size, dtype=np.intp)
        values = np.empty(size, dtype=np.float64)
        self._to_arrays(keys, values)
        return keys, values
```

Reply:
This works. Thanks!